### PR TITLE
Upgrade major dependencies

### PR DESCRIPTION
**Pull request by ericsoderberghp**
Upgraded major dependencies.
- eslint upgrade led to some quoting changes
- migrated unit tests to @testing-library/react

### CODE REVIEW

**Review by jcfilben — COMMENTED** [review]
When I run storybook the addon knobs is no longer showing up. Also the storybook files config.js and addon.js are marked as deprecated https://github.com/storybookjs/storybook/blob/master/MIGRATION.md#to-mainjs-configuration

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "release-stable": "babel-node ./tools/release-stable",
     "lint": "eslint src",
     "storybook": "start-storybook -p 9002 -c storybook",
-    "build-storybook": "build-storybook -c storybook -o storybook-static -s ./storybook/public",
```
Comment (jcfilben) [inline package.json:30]: Just curious what removing `-s ./storybook/public` is doing

File: webpack.config.babel.js [hunk 2]
```diff
@@ -29,11 +31,7 @@ export default {
     extensions: ['.js', '.json'],
   },
   plugins,
-  node: {
-    fs: 'empty',
-    net: 'empty',
-    tls: 'empty',
-  },
+  node: false,
```
Comment (halocline) [inline webpack.config.babel.js:34]: No concern here, just linking documentation for this setting.

https://webpack.js.org/configuration/node/#node

**Review by halocline — APPROVED** [review]
- [x] config changes
- [x] dev dependencies compatible
- [x] build successful
- [x] Jest snapshot changes expected
- [x] linter settings applied are appropriate

File: package.json (hunk 1 above)
Comment (halocline) [reply package.json:30]: I believe it is simply removing the directory since it is unused / isn't being populated with any static assets.

Comment (ericsoderberghp) [reply package.json:30]: Correct. With the latest storybook, it was failing because it couldn't read the directory.

Comment (jcfilben) [reply package.json:30]: Great thanks for the explanation

**Review by jcfilben — APPROVED** [review]

**Pull request merged**